bulletroboy/exoforce.py

```diff
@@ -268,7 +270,13 @@ def __init__(self, cage_conf, operator):
 								for muscle in cage_conf.muscle_units ]
 		self.cage = Cage(cage_conf.cage_structure['height'], cage_conf.cage_structure['radius'], self.get_motors())
 
-	def update(self, new_angle, motor_forces):
+	def update(self, new_angle, motor_forces, automatic_cage_rotation):
+
+		if automatic_cage_rotation == True:
```

Comment:
- [ ] make `automatic_cage_rotation` a field in this class
- [ ] pass it as a parameter in the init
- [ ] set a default value for it (False)
- [ ] remove it from the update fucntion signature

Reply:
I deleted this implementation because it does not match the current status of the code. I will use a different method (but not in this branch)